integrations/langchain4j/providers/open-ai/src/main/java/io/helidon/integrations/langchain4j/providers/openai/OpenAiStreamingChatModelFactory.java

```diff
@@ -0,0 +1,142 @@
+     */
+    OpenAiStreamingChatModelFactory(@Service.Named(OpenAi.STREAM_CHAT_MODEL)
+                                    Supplier<Optional<Tokenizer>> openAiChatModelTokenizer,
+                                    @Service.Named(OpenAi.OPEN_AI) Supplier<Optional<Tokenizer>> openAiTokenizer,
+                                    Supplier<Optional<Tokenizer>> tokenizer,
+                                    @Service.Named(OpenAi.STREAM_CHAT_MODEL) Supplier<Optional<Proxy>> openAiChatModelProxy,
+                                    @Service.Named(OpenAi.OPEN_AI) Supplier<Optional<Proxy>> openAiProxy,
+                                    Supplier<Optional<Proxy>> proxy,
+                                    Config config) {
+        var configBuilder = OpenAiStreamingChatModelConfig.builder().config(config.get(OpenAiChatModelConfig.CONFIG_ROOT));
+
+        this.model = () -> buildModel(configBuilder,
+                                      openAiChatModelTokenizer,
+                                      openAiTokenizer,
+                                      tokenizer,
+                                      openAiChatModelProxy,
+                                      openAiProxy,
+                                      proxy);
+    }
+
+    /**
+     * Create the OpenAI model from its configuration.
+     *
+     * @param config configuration to use
+     * @return a new model instance
+     * @throws java.lang.IllegalStateException in case the configuration is not enabled
+     */
+    public static OpenAiStreamingChatModel create(OpenAiStreamingChatModelConfig config) {
+        if (!config.enabled()) {
+            throw new IllegalStateException("Cannot create a model when the configuration is disabled.");
+        }
+        var builder = OpenAiStreamingChatModel.builder();
+        config.baseUrl().ifPresent(builder::baseUrl);
+        config.apiKey().ifPresent(builder::apiKey);
+        config.organizationId().ifPresent(builder::organizationId);
+        config.modelName().ifPresent(builder::modelName);
+        config.temperature().ifPresent(builder::temperature);
+        config.topP().ifPresent(builder::topP);
+        if (!config.stop().isEmpty()) {
+            builder.stop(config.stop());
+        }
+        config.maxTokens().ifPresent(builder::maxTokens);
+        config.maxCompletionTokens().ifPresent(builder::maxCompletionTokens);
+        config.presencePenalty().ifPresent(builder::presencePenalty);
+        config.frequencyPenalty().ifPresent(builder::frequencyPenalty);
+        if (!config.logitBias().isEmpty()) {
+            builder.logitBias(config.logitBias());
+        }
+        config.responseFormat().ifPresent(builder::responseFormat);
+        config.seed().ifPresent(builder::seed);
+        config.user().ifPresent(builder::user);
+        config.strictTools().ifPresent(builder::strictTools);
+        config.parallelToolCalls().ifPresent(builder::parallelToolCalls);
```

Comment:
I am missing few props from lc4J api:
- store
- metadata
- serviceTier

Are those omitted intentionally?

Reply:
They are not on the builder.

Reply:
Beta 1 was released yesterday